src/components/PasteRestrictedBox.tsx

```diff
@@ -18,14 +18,24 @@ export default function PasteRestrictedBox({
   onChange
 }: PasteRestrictedBoxProps) {
   const [value, setValue] = useState<string>("");
+  const [disableInput, setDisableInput] = useState<boolean>(true)
 
   const handlePaste = async () => {
-    await navigator.clipboard.readText().then(
-      (text) => {
-        setValue(text)
-        onChange?.(text)
-      }
-    )
+    try {
+      await navigator.clipboard.readText().then(
+        (text) => {
+          setValue(text)
+          onChange?.(text)
+        }
+      )
+    } catch (error) {
```

Comment:
Cool. That looks good, So it will not block users with other browser that doesn't support Paste 👍